contracts/src/BorrowerOperations.sol

```diff
@@ -15,6 +15,7 @@ import "./Dependencies/LiquityBase.sol";
 import "./Dependencies/AddRemoveManagers.sol";
 import "./Types/LatestTroveData.sol";
 import "./Types/LatestBatchData.sol";
+import "./BatchManagerOperations.sol";
```

Comment:
Not part of this PR but lets update the comment below regarding SystemParams, I guess it was before we decided to call it everywhere?

```
However, to reduce contract size, the following parameter is read directly from SystemParams when needed:
 * - SCR: Only used in shutdown() function
 ```